### PR TITLE
Gulp with npm publish support

### PR DESCRIPTION
**Pull request by jorkas**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -57,17 +57,13 @@ To create a new version, there are multiple steps:
 * First, we want to increase the version number in the `package.json` and `bower.json` files and commit this.
 * Then we want to tag the commit, because `bower` use the git tag as version number instead of reading a manifest file.
 * To release the new version, we will just push those changes to master
-* And last but not least, now want to update the doc (style-guide) too, use the `gulp gh-pages` task for this.
+* Publish to npm with `npm publish`
```
Comment (bzf) [inline README.markdown]: Shouldn't this be `gulp publish`? :)

Comment (jorkas) [inline README.markdown]: @bzf True, i was thinking more about what it actually does instead IF the gulp task. Maybe just write publish the version to npm instead?

Comment (bzf) [inline README.markdown]: Ah, it's a description of what the script does. Then the "publish the version to npm" sounds better imo 😄 👍

**Comment by dbackeus** [comment]
👍

**Pull request merged**